en/developer_guide/backbone_routing.rst

```diff
@@ -35,8 +32,7 @@ In a configuration file, you could add a route to the router:
         });
     })(window.appRouter);
 
-When you declare a new route in Backbone, you should first give the route pattern for the route generation.
-Then you should declare the method used for the matching.
+When you declare a new route in Backbone, you should declare the method used for the matching.
```

Comment:
I think this line became useless here at the moment the previous one was removed. You could remove it and add a modified one before the code sample to introduce it. Something like ``To declare a new route in Backbone, you have to declare the method run when the route is triggered :``